### ISSUE TITLE
Can't enable emoji pack with new name

### ISSUE THREAD

**Issue by pcroland** [mintty/wsltty#321]
I recently got a 4K monitor and because of the high-DPI i would like to use 32x32 emojis but the config sets `none` as `Emojis` if the images in the folder are not 15x15.

**Comment by mintty** [comment]
There is no limit to emoji size. Those deployed via the `getemojis` tool are 72x72. I also have 128x128 Joypixels installed and they work fine.

**Comment by mintty** [comment]
Please describe the contents of your emojis config directory (location, subdirectories, how you deployed files...).

**Comment by pcroland** [comment]
I used the getemojis script, fetched the images to a folder called `apple_new`. The folder is at `%appdata%/wsltty/emojis/`.
The config looks like this:
```
# To use common configuration in %APPDATA%\mintty, simply remove this file
ThemeFile=moe-dark-darker.minttyrc
Font=FiraCode Nerd Font Mono
CursorType=line
OpaqueWhenFocused=yes
Transparency=off
Term=xterm-256color
BellFile=ambient-drop
FontHeight=11
FontSmoothing=partial
Columns=200
Rows=55
Scrollbar=right
Language=en_US
BoldAsFont=yes
LigaturesSupport=2
Ligatures=2
BoldAsColour=no
EmojiPlacement=middle
Emojis=none
```
It won't change `Emojis` to `apple_new`  and it sets `none`, I can only set `apple` which contains the 15x15 icons.

**Comment by pcroland** [comment]
Here's a video of the behaviour: [vid](https://envs.sh/Qxj.mp4)

**Comment by mintty** [comment]
The directory must be called "apple", not "apple_new". Only the predefined names are supported.

**Comment by pcroland** [comment]
Oh, thank you. I would suggest filtering the names in the GUI selection to make it less confusing.

**Comment by pcroland** [comment]
Also, some new emojis aren't displaying despite having them in the folder with proper names.

**Comment by mintty** [comment]
I may check whether to clarify the description. I had thought about listing emoji directories dynamically. The thing is, emoji file name patterns are different among the sets and there is some hard-coded adaptation to it in mintty, so the effort isn't justified, given that new emoji sets do not arise often.

**Comment by mintty** [comment]
Which emojis are not supported? Mintty 3.6.3 supports some new emojis, wsltty release is still at 3.6.1.

**Comment by pcroland** [comment]
Pregnant man for example :D

**Comment by mintty** [comment]
That's U+1FAC3, right? Do you an emoji image file for it in your emoji config directory? I don't.

**Comment by pcroland** [comment]
Yeah, I have them in my folder:
![img](https://telegra.ph/file/a337ea9eb2b0abe1d79e1.png)
If I paste the emoji into WSLtty it shows this: `<0001fac3>`

**Comment by mintty** [comment]
It shows up for me in 3.6.3, so please wait for a new wsltty release. I'll make one soon.

**Comment by pcroland** [comment]
Okay, thanks. Wouldn't it be possible to show every emoji that is present in the folder automatically? That way you could update the folder without updating mintty/wsltty.

**Comment by mintty** [comment]
This would perhaps be possible for new emoji sequences, as they are formed to similar patterns.
For new distinct emojis, however, it is a problem as previous Unicode versions do not define additional emojis of later versions.

**Comment by mintty** [comment]
... Furthermore emojis have properties that are derived from Unicode lists and converted into tables. E.g. if an emoji arises for the letter A (as there are emojis for digits 0...9), you wouldn't want to appear letter A as an emoji _by default_.

**Comment by mintty** [comment]
Released 3.6.4, supporting dynamic flags emojis.

**Comment by mintty** [comment]
Assuming this issue is resolved, closing.
